fury/actor.py

```diff
@@ -2506,6 +2506,32 @@ def texture(rgb, interp=True):
     return act
 
 
+def update_video_texture(grid, arr):
+    """
+    Updates texture of an actor by updating the vtkImageData
+    assigned to the vtkTexture object.
+
+    Parameters
+    ----------
+    grid: vtkImageData
```

Comment:
Please change this to: 
``texture_actor: texture_actor or image_data 
Check the instance and if actor then call texture_actor.GetTexture().GetInput() otherwise assume input already vtkImageData.